website/.prettierrc

```diff
@@ -20,12 +20,11 @@
             }
         },
         {
-            "files": "*.md",
+            "files": ["package.json", "package-lock.json"],
```

Comment:
Can you explain this change? It used to be so that `prettier` would only be applied to `md` files? But you've extended that to more file types in https://github.com/explosion/spaCy/pull/11996?

Reply:
Looks like this change is just applies specific formatting rules to those two JSON files. The diff here is potentially a bit misleading because it combined the addition of the new rule and the removal of an old rule (`.md` is not needed anymore, our files are all `.mdx` now).

Reply:
Sometimes git just finds the most co fusing way to display a diff 😅